frontend/challedu-app/src/components/navbar.js

```diff
@@ -0,0 +1,23 @@
+import React, { Component } from "react";
+import { Button , ButtonGroup  }  from 'reactstrap';
+import 'bootstrap/dist/css/bootstrap.min.css';
+import '../App.css';
+import logo from '../images/logo.svg';
+
+export default class navbar extends Component {
+    render() {
+        return (
+            <div id="navbar">
```

Comment:
this chouls be a nav bar => https://getbootstrap.com/docs/4.0/components/navbar/

Reply:
solved